app/content/react/library-reference/widgets.md

```diff
@@ -58,9 +59,31 @@ Renders a `<CategoryWidget />` component, binded to a source at redux. The widge
       onError={console.error}
     />
   );
-
+  
   // The operationColumn wouldn't be required if using AggregationTypes.COUNT, to count the number of countries per continent
   ```
+  
+  In case you have the `population` disaggregated by gender: `population_m` and `population_f`, you can use an array in `operationColumn` and sum them into a single column using `joinOperation`. By this way, you avoid to have duplicated data in your dataset and you can compose widgets using multiple columns.
+  
+  ```js
+  import { CategoryWidget } from "@carto/react-widgets";
+  
+  const customFormatter = (value) => `${value} people`;
+  
+  return (
+    <CategoryWidget
+      id="populationByContinent"
+      title="Population by continent"
+      dataSource="countriesSourceId"
+      column="continent"
+      operationColumn={["population_m", "population_f"]}
+  		joinOperation={AggregationTypes.SUM}
```

Comment:
Indentation seems to be wrong and we should probably add AggregationTypes to the imports above.